packages/manager/modules/pci/src/components/project/storages/databases/engines.constants.js

```diff
@@ -11,7 +11,7 @@ export const ENGINES_NAMES = {
   mongodb: 'MongoDB',
   postgresql: 'PostgreSQL',
   kafka: 'Kafka',
-  redis: 'Redis',
+  redis: 'Redis™*',
```

Comment:
As stated bellow by @frenautvh , could the asterisk be put outside of the engine's name ?

Reply:
It is complicated because the same variable is repeated in several places. I've gone for the simplest